### PR TITLE
Front end landing page

### PR DESCRIPTION
**Pull request by dorisgjata**
As of right now it doesn't have much. Just 3 tabs to paginate through the collection, tour and favorites.
You can browse the collection from the dummy data.
Need to figure out how to preload the data on the collection tab since that's the one that's initially selected.

I've added redux, react-router and mui for the front-end. Redux has some new stuff called redux-toolkit which simplifies defining the store/reduce/actions.

### CODE REVIEW

File: src/react/src/features/collection/Collection.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  return (
+    <Grid container spacing={2} justifyContent="center" >
+      {collection.map((artwork, index) => {
+        return (
+          <Grid item xs={11}>
+            <Card key={index} variant="outlined" style={{ paddingBottom: 2 }}>
+              <CardMedia
+                component="img"
+                sx={{ width: 200 }}
```
Comment (sbracellari) [inline src/react/src/features/collection/Collection.tsx]: what is this prop? I've never seen it

Comment (dorisgjata) [reply src/react/src/features/collection/Collection.tsx]: the sx? It's a new way of styling in mui
https://mui.com/system/the-sx-prop/
https://stackoverflow.com/questions/52124938/how-to-add-padding-and-margin-to-all-material-ui-components

File: src/react/src/features/collection/Collection.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          Carnegie Museum of Art is arguably the first museum of contemporary art in the United States, collecting the “Old Masters of tomorrow” since the inception of the Carnegie International in 1896.
+          Our collection of more than 34,000 objects features a broad spectrum of visual art, including painting and sculpture; prints and drawings; photographs; architectural casts, renderings, and models; decorative arts and design; and film, video, and digital imagery. The museum also houses the archive of over 70,000 negatives by photographer Charles “Teenie” Harris.
+        </Typography>
+        <Box sx={{ display: 'flex', justifyContent: 'center', pb: 2 }}>
+          <Button onClick={() => dispatch(getCollection())} variant="contained">View collection</Button>
+        </Box>
+      </Grid>
+      {collection.map((artwork, index) => {
+        return (
+          <Grid key={`${index}_${artwork.title}`} item xs={11}>
```
Comment (sbracellari) [inline src/react/src/features/collection/Collection.tsx:24]: do we want to have a grid for each artwork? what is the intention behind this

Comment (dorisgjata) [reply src/react/src/features/collection/Collection.tsx:24]: It's a grid item not grid container tho to keep the size of the cards uniform. It makes sense to have it for every card and we could add breakpoints for the other screen sizes as well.

**Pull request merged**